gsy_myco_sdk/setups/myco_matcher.py

```diff
@@ -16,31 +17,40 @@ class MycoMatcher(base_matcher):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.is_finished = False
+        self.request_area_id_name_map()
+        self.id_list = []
+
+    def on_area_map_response(self, data):
+        market_list = ["Community"]
+        for market in market_list:
```

Comment:
Minor, but how about having this as an optional parameter of the constructor and saving it into self.
This allows you to pass the market_list on intialisation of the MycoMatcher(). 
```suggestion
    def __init__(self, market_list=None, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.is_finished = False
        self.request_area_id_name_map()
        self.id_list = []
        self.market_list = market_list or []

    def on_area_map_response(self, data):
        for market in self.market_list:
```

Reply:
I would probably stick with the original proposal, if that is ok!

Reply:
Too bad. Just keep this best practice in mind for the refactoring of the trading scripts. Thanks!